doc/languages-frameworks/python.section.md

```diff
@@ -1049,12 +1049,12 @@ If you want the whole of Nixpkgs to use your modifications, then you can use
 ```nix
 let
   pkgs = import <nixpkgs> {};
-  newpkgs = import pkgs.path { overlays = [ (pkgsself: pkgssuper: {
-    python27 = let
-      packageOverrides = self: super: {
-        numpy = super.numpy_1_10;
+  newpkgs = import pkgs.path { overlays = [ (self: super: {
+    python38 = let
+      packageOverrides = python-self: python-super: {
+        numpy = python-super.numpy_1_18.3;
```

Comment:
Typo
```
python-super.numpy_1_18.3
```
Point is incorrect, and this package does not exist. Note also that, if we add a version, we try to be as unspecific as possible. Patch version you typically do not want to pin to, but major or minor you would. This, `numpy_1_18` is a better value.

Reply:
Argh, you're right.  I also don't particularly like this example, as *most of the time* we just have 1 version of a package and the "real" way you have to do it is by using `overrideAttrs` to re-fetch pypi and your `src`.  

Ideally we'd either make it the more heavyweight example (with the full `src` fetch on the chosen version), or swap it to an example showing how to do something like build pytorch with cuda (except that doesn't work at the moment and is proprietary).

Reply:
overridePythonAttrs ;-) https://github.com/NixOS/rfcs/pull/67